### PR TITLE
Create a blueprint backed item or header without needing to define your own ItemContent

### PR DESCRIPTION
**Pull request by kyleve**
A common ask is for a way to display rows without needing to create entirely new `BlueprintItemContent` instances without needing to define a whole new type. This PR lets you do this for headers and footers:

```
section += podcasts.map { podcast in
    
    Item(podcast, identifier: \.name) { _, _ in
        PodcastElement(podcast: podcast)
    } background: { _, _ in
        Box(backgroundColor: .white, cornerStyle: .square)
    } selectedBackground: { _, _ in
        Box(backgroundColor: .lightGray, cornerStyle: .square)
    }
}
```

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
```
Comment (kyleve) [inline Gemfile:1]: Intentional changes in here; no reason to pin these during `bundle update`.

File: Listable/Sources/Item/Item.swift [hunk 2]
```diff
@@ -77,6 +77,9 @@ public struct Item<Content:ItemContent> : AnyItem
     
     public typealias Build = (inout Item) -> ()
```
Comment (kyleve) [inline Listable/Sources/Item/Item.swift:78]: Document me

**Review by kylebshr — APPROVED** [review]
LGTM, sorry this went unnoticed for so long

File: ListableUI/Sources/Item/Item.swift [hunk 3]
```diff
@@ -81,6 +81,9 @@ public struct Item<Content:ItemContent> : AnyItem
```
Comment (kyleve) [inline ListableUI/Sources/Item/Item.swift:81]: Revert these changes

**Comment by kyleve** [comment]
Thanks! No rush, not an urgent one.
